src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.operation.BalanceHandler;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.PurchaseHandler;
+import core.basesyntax.operation.ReturnHandler;
+import core.basesyntax.operation.SupplyHandler;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+    private ShopService shopService;
+    private OperationStrategy operationStrategy;
+    private Map<FruitTransaction.Operation, OperationHandler> handlerMap;
+
+    @BeforeEach
+    void setUp() {
+        handlerMap = new HashMap<>();
+        handlerMap.put(FruitTransaction.Operation.RETURN, new ReturnHandler(shopService));
+        handlerMap.put(FruitTransaction.Operation.PURCHASE, new PurchaseHandler(shopService));
+        handlerMap.put(FruitTransaction.Operation.SUPPLY, new SupplyHandler(shopService));
+        handlerMap.put(FruitTransaction.Operation.BALANCE, new BalanceHandler(shopService));
+        operationStrategy = new OperationStrategyImpl(handlerMap);
+    }
+
+    @Test
+    void testGetOperationHandler_forReturn() {
+        OperationHandler returnOperation = operationStrategy.get(FruitTransaction.Operation.RETURN);

```

Comment:
same comment here, as I understand you test thet method get from Map works well? 
If I am wrong, let's discuss in chat ))

Reply:
Looking at the class (OperationStrategyImplTest) as a whole, I check that the handler for each operation is returning correctly and that the handlers are correctly assigned, using the get method